modules/javafx.graphics/src/main/java/com/sun/prism/impl/ps/PaintHelper.java

```diff
@@ -307,7 +307,7 @@ private static void setMultiGradient(Shader shader,
         stopVals.rewind();
         shader.setConstants("fractions", stopVals, 0, MULTI_MAX_FRACTIONS);
         float index_y = initGradient(paint);
-        shader.setConstant("offset", index_y / (float)MULTI_CACHE_SIZE + HALF_TEXEL_Y);
+        shader.setConstant("offset", index_y / MULTI_CACHE_SIZE + HALF_TEXEL_Y);
```

Comment:
Here's another.

Reply:
index_y is a float, so the typecast is unnecessary

Reply:
Right. In this case I don't really see a problem.